tools/AzureRM.BootStrapper/AzureRM.Bootstrapper.Format.ps1xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Configuration>
+  <ViewDefinitions>
+    <View>
+      <Name>ProfileMapDataView</Name>
+      <ViewSelectedBy>
+        <TypeName>ProfileMapData</TypeName>
+      </ViewSelectedBy>
+      <ListControl>
+        <ListEntries>
+          <ListEntry>
+            <ListItems>
+              <ListItem>
+                <PropertyName>ProfileName</PropertyName>
+              </ListItem>
+              <ListItem>
+                <Label>Modules</Label>
+                <ScriptBlock> 
+                  $RequiredModules = "AzureRM", "AzureRM.Storage", "AzureRM.Compute", "AzureRM.Network"
```

Comment:
If we are going to choose modules, these should be ```AzureRm```, ```AzureRM.Compute```, ```AzureRM.Websites```, ```AzureRM.Storage```

Network is important, but only insofar as it supports Compute, so no reason to call it out here